Vagrantfile

```diff
@@ -58,7 +58,7 @@ Vagrant.configure('2') do |config|
     config.hostmanager.aliases = hostnames + redirects
   elsif Vagrant.has_plugin?('landrush') && trellis_config.multisite_subdomains?
     config.landrush.enabled = true
-    config.landrush.tld = config.vm.hostname
+    config.landrush.tld = trellis_config.site_hosts_canonical.select { |host| !host[/\.#{main_hostname}$/] }
```

Comment:
Minor but you can use `reject` without the negative operator. Also `end_with`:

```ruby
trellis_config.site_hosts_canonical.reject { |host| host.end_with?(".#{main_hostname}") }
```

Reply:
Awesome!